doc/LIMITS.md

```diff
@@ -106,87 +136,90 @@ libp2p stream multiplexers impose limits on the amount of streams that can be op
 
 These settings are done on a per-muxer basis, please see the README of the relevant muxer you are using.
 
+All fields are optional. The default values are defined in [@libp2p/mplex/src/mplex.ts](https://github.com/libp2p/js-libp2p-mplex/blob/master/src/mplex.ts) - please see that file for the current values.
```

Comment:
This should be moved under the Mplex section.

Reply:
Done